exporter/metric/metric.go

```diff
@@ -255,16 +261,19 @@ func (me *metricExporter) createMetricDescriptorIfNeeded(ctx context.Context, md
 
 // exportTimeSeriees create TimeSeries from the records in cps.
 // res should be the common resource among all TimeSeries, such as instance id, application name and so on.
-func (me *metricExporter) exportTimeSeries(ctx context.Context, cps export.CheckpointSet) error {
+func (me *metricExporter) exportTimeSeries(ctx context.Context, res *resource.Resource, ilr export.InstrumentationLibraryReader) error {
 	tss := []*monitoringpb.TimeSeries{}
 
-	aggError := cps.ForEach(export.CumulativeExportKindSelector(), func(r export.Record) error {
-		ts, err := me.recordToTspb(&r)
-		if err != nil {
-			return err
-		}
-		tss = append(tss, ts)
-		return nil
+	// TODO do something with the Library?
```

Comment:
Consider opening issues to track these TODOs (and mentioning the issue in the TODO).